openmetadata-ui/src/main/resources/ui/src/styles/app.less

```diff
@@ -476,3 +476,21 @@ a[href].link-text-grey,
   height: @entity-details-tab-height;
   overflow-y: scroll;
 }
+
+/* Diff style */
+
+.diff-added {
```

Comment:
@karanh37 do you remember why we haven't add this globally?

Reply:
It was there globally but was part of temp.css(which we are going to remove).
I asked @Ashish8689 to move it to app.less and use variables.